Firestore/Source/Public/FirebaseFirestore/FIRFirestore.h

```diff
@@ -163,6 +163,12 @@ NS_SWIFT_NAME(Firestore)
 
 #pragma mark - Network
 
+/**
+ * Configures Firestore to connect to an emulated host instead of the default remote backend. Once
+ * Firestore is initialized, this value cannot be changed.
```

Comment:
What's the startup lifecycle like here? This method must be called after `Firestore.firestore()` is called, which is what I would think means "Firestore is initialized" at that point. FWIW I don't have an issue with the method itself, mostly the docs.

Reply:
On other platforms we say "after Firestore has been used" or similar.

Reply:
Amended wording for better clarity.